src/brokers/utils/add-broker.ts

```diff
@@ -129,6 +169,10 @@ export enum ArtemisReducerOperations {
   updateAcceptorFactoryClass,
   /** Updates the configuration's factory Class */
   updateConnectorFactoryClass,
+  /** updates the current state item */
+  updateEditorState,
+  /** updates the whole model */
+  setModel,
```

Comment:
The enumeration is alphabetically ordered, can you place this `setModel` before ` setReplicasNumber` please?